examples/pom.xml

```diff
@@ -0,0 +1,190 @@
+
```

Comment:
You can add the checkstyle plugin to this pom file exactly the same as it has been done in `java/pom.xml` then you can run `mvn package -DskipTests -P checkstyle` to look for errors. 
I expect there will be a few errors as the rules were taken from Helidon (modified slightly by Aleks) and they are very strict.
There is a code style xml file that you can import into IntelliJ `etc/intellij-codestyle.xml` but it does not pick up everything (no @author tags allowed, no trailing spaces on lines, blank lines in imports, packages need package-info.java etc...)

Reply:
fixed

Reply:
fixed